2024-01-06 14:28:15,740 INFO    StreamThr :62535 [internal.py:wandb_internal():86] W&B internal server running at pid: 62535, started at: 2024-01-06 14:28:15.738655
2024-01-06 14:28:15,741 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status
2024-01-06 14:28:15,741 INFO    WriterThread:62535 [datastore.py:open_for_write():85] open: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/run-wf6xw9ej.wandb
2024-01-06 14:28:15,743 DEBUG   SenderThread:62535 [sender.py:send():382] send: header
2024-01-06 14:28:15,743 DEBUG   SenderThread:62535 [sender.py:send():382] send: run
2024-01-06 14:28:16,051 INFO    SenderThread:62535 [dir_watcher.py:__init__():211] watching files in: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files
2024-01-06 14:28:16,052 INFO    SenderThread:62535 [sender.py:_start_run_threads():1136] run started: wf6xw9ej with start time 1704580094.965301
2024-01-06 14:28:16,064 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: check_version
2024-01-06 14:28:16,064 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: check_version
2024-01-06 14:28:16,205 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: run_start
2024-01-06 14:28:16,206 DEBUG   HandlerThread:62535 [system_info.py:__init__():32] System info init
2024-01-06 14:28:16,207 DEBUG   HandlerThread:62535 [system_info.py:__init__():47] System info init done
2024-01-06 14:28:16,207 INFO    HandlerThread:62535 [system_monitor.py:start():194] Starting system monitor
2024-01-06 14:28:16,207 INFO    SystemMonitor:62535 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-01-06 14:28:16,207 INFO    HandlerThread:62535 [system_monitor.py:probe():214] Collecting system info
2024-01-06 14:28:16,207 DEBUG   HandlerThread:62535 [system_info.py:probe():196] Probing system
2024-01-06 14:28:16,207 INFO    SystemMonitor:62535 [interfaces.py:start():190] Started cpu monitoring
2024-01-06 14:28:16,209 INFO    SystemMonitor:62535 [interfaces.py:start():190] Started disk monitoring
2024-01-06 14:28:16,209 INFO    SystemMonitor:62535 [interfaces.py:start():190] Started gpuapple monitoring
2024-01-06 14:28:16,209 INFO    SystemMonitor:62535 [interfaces.py:start():190] Started memory monitoring
2024-01-06 14:28:16,210 INFO    SystemMonitor:62535 [interfaces.py:start():190] Started network monitoring
2024-01-06 14:28:16,213 DEBUG   HandlerThread:62535 [gitlib.py:_init_repo():56] git repository is invalid
2024-01-06 14:28:16,214 DEBUG   HandlerThread:62535 [system_info.py:probe():244] Probing system done
2024-01-06 14:28:16,214 DEBUG   HandlerThread:62535 [system_monitor.py:probe():223] {'os': 'macOS-13.6-arm64-arm-64bit', 'python': '3.9.16', 'heartbeatAt': '2024-01-06T22:28:16.207812', 'startedAt': '2024-01-06T22:28:10.797940', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'host': 'CurtisMac.local', 'username': 'curtispond', 'executable': '/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/bin/python', 'cpu_count': 10, 'cpu_count_logical': 10, 'disk': {'/': {'total': 460.4317207336426, 'used': 165.75806045532227}}, 'gpuapple': {'type': 'arm', 'vendor': 'Apple'}, 'memory': {'total': 16.0}}
2024-01-06 14:28:16,214 INFO    HandlerThread:62535 [system_monitor.py:probe():224] Finished collecting system info
2024-01-06 14:28:16,214 INFO    HandlerThread:62535 [system_monitor.py:probe():227] Publishing system info
2024-01-06 14:28:16,214 DEBUG   HandlerThread:62535 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-01-06 14:28:16,215 DEBUG   HandlerThread:62535 [system_info.py:_save_pip():68] Saving pip packages done
2024-01-06 14:28:16,215 DEBUG   HandlerThread:62535 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-01-06 14:28:17,057 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/requirements.txt
2024-01-06 14:28:17,058 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/conda-environment.yaml
2024-01-06 14:28:21,602 DEBUG   HandlerThread:62535 [system_info.py:_save_conda():87] Saving conda packages done
2024-01-06 14:28:21,603 INFO    HandlerThread:62535 [system_monitor.py:probe():229] Finished publishing system info
2024-01-06 14:28:21,603 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:28:21,603 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:28:21,603 DEBUG   SenderThread:62535 [sender.py:send():382] send: files
2024-01-06 14:28:21,603 INFO    SenderThread:62535 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-01-06 14:28:21,610 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:28:21,610 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:28:21,610 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:28:21,827 DEBUG   SenderThread:62535 [sender.py:send():382] send: telemetry
2024-01-06 14:28:22,081 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/conda-environment.yaml
2024-01-06 14:28:22,081 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/wandb-metadata.json
2024-01-06 14:28:22,100 INFO    wandb-upload_0:62535 [upload_job.py:push():131] Uploaded file /var/folders/02/t1w3xsh916z60p5jhmd5xbc00000gn/T/tmph6tufajiwandb/hfr5tzfy-wandb-metadata.json
2024-01-06 14:28:26,843 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:28:31,864 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:28:36,614 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:28:36,616 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:28:36,617 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:28:37,734 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:28:42,751 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:28:47,772 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:28:48,201 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/config.yaml
2024-01-06 14:28:51,617 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:28:51,619 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:28:51,619 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:28:53,787 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:28:58,797 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:29:03,809 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:29:06,627 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:29:06,630 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:29:06,632 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:29:08,907 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:29:13,926 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:29:16,215 DEBUG   SystemMonitor:62535 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-01-06 14:29:16,218 DEBUG   SenderThread:62535 [sender.py:send():382] send: telemetry
2024-01-06 14:29:16,219 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:29:19,234 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:29:19,357 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/config.yaml
2024-01-06 14:29:20,362 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/config.yaml
2024-01-06 14:29:21,627 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:29:21,628 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:29:21,634 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:29:24,825 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:29:29,845 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:29:34,864 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:29:36,631 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:29:36,632 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:29:36,636 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:29:40,764 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:29:45,783 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:29:46,226 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:29:51,251 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:29:51,634 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:29:51,635 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:29:51,639 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:29:56,841 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:30:01,860 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:30:06,638 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:30:06,639 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:30:06,643 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:30:07,749 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:30:12,774 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:30:16,233 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:30:18,242 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:30:21,640 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:30:21,643 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:30:21,644 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:30:23,824 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:30:28,844 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:30:33,858 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:30:36,640 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:30:36,643 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:30:36,649 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:30:39,791 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:30:44,802 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:30:46,243 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:30:46,331 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:30:46,758 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:30:48,772 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:30:50,350 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:30:51,644 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:30:51,645 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:30:51,652 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:30:55,846 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:30:58,436 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:31:00,838 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:31:01,455 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:31:06,465 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:31:06,646 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:31:06,648 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:31:06,656 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:31:10,456 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:31:12,469 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:31:12,898 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:31:16,244 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:31:18,254 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:31:21,651 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:31:21,653 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:31:21,657 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:31:23,850 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:31:26,329 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:31:28,976 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:31:29,341 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:31:34,363 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:31:36,655 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:31:36,655 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:31:36,660 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:31:38,515 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:31:39,520 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:31:41,021 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:31:44,543 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:31:46,250 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:31:50,268 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:31:51,660 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:31:51,661 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:31:51,665 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:31:55,789 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:31:56,335 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:31:57,091 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:32:01,356 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:32:06,381 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:32:06,583 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:32:06,659 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:32:06,659 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:32:06,666 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:32:09,150 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:32:11,777 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:32:16,258 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:32:17,265 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:32:21,662 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:32:21,662 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:32:21,670 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:32:22,833 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:32:26,344 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:32:28,355 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:32:29,253 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:32:33,373 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:32:36,671 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:32:36,673 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:32:36,677 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:32:38,664 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:32:38,665 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:32:41,308 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:32:43,686 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:32:46,265 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:32:49,277 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:32:51,672 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:32:51,674 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:32:51,681 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:32:54,872 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:32:56,350 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:32:57,379 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:33:00,369 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:33:05,385 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:33:06,677 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:33:06,679 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:33:06,684 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:33:06,744 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:33:09,446 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:33:10,776 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:33:15,796 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:33:16,274 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:33:21,291 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:33:21,684 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:33:21,686 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:33:21,694 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:33:26,909 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:33:31,930 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:33:36,690 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:33:36,692 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:33:36,699 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:33:36,949 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:33:41,972 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:33:46,290 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:33:46,377 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:33:47,384 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:33:47,609 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:33:51,691 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:33:51,693 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:33:51,700 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:33:52,876 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:33:57,897 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:34:02,913 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:34:06,699 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:34:06,701 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:34:06,707 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:34:08,888 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:34:13,905 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:34:16,288 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:34:16,382 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:34:17,751 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:34:19,396 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:34:21,702 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:34:21,707 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:34:21,708 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:34:24,943 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:34:29,968 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:34:34,981 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:34:36,706 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:34:36,707 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:34:36,708 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:34:40,926 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:34:45,939 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:34:46,291 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:34:46,373 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:34:47,891 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:34:51,396 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:34:51,713 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:34:51,715 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:34:51,717 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:34:56,851 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:35:01,864 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:35:06,712 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:35:06,712 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:35:06,716 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:35:07,856 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:35:12,882 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:35:16,296 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:35:16,368 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:35:17,027 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:35:18,377 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:35:21,717 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:35:21,718 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:35:21,719 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:35:23,856 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:35:28,876 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:35:33,895 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:35:36,725 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:35:36,727 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:35:36,727 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:35:39,007 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:35:44,026 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:35:46,305 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:35:46,383 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:35:47,153 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:35:49,392 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:35:51,727 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:35:51,728 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:35:51,729 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:35:54,883 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:35:59,895 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:36:04,919 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:36:06,732 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:36:06,734 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:36:06,734 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:36:10,844 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:36:15,860 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:36:16,309 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:36:16,391 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:36:17,299 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:36:21,415 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:36:21,734 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:36:21,735 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:36:21,735 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:36:26,857 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:36:31,881 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:36:36,741 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:36:36,742 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:36:36,743 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:36:37,867 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:36:42,882 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:36:46,317 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:36:46,414 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:36:47,433 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:36:48,422 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:36:51,746 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:36:51,747 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:36:51,749 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:36:53,927 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:36:58,948 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:37:03,964 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:37:06,746 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:37:06,747 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:37:06,748 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:37:09,855 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:37:14,878 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:37:16,323 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:37:16,408 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:37:17,585 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:37:20,424 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:37:21,753 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:37:21,754 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:37:21,757 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:37:25,895 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:37:30,910 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:37:35,932 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:37:36,754 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:37:36,756 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:37:36,757 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:37:40,935 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:37:45,954 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:37:46,330 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:37:46,409 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:37:47,747 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:37:51,430 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:37:51,756 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:37:51,761 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:37:51,762 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:37:56,879 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:38:01,895 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:38:06,758 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:38:06,765 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:38:06,766 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:38:07,885 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:38:12,907 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:38:16,332 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:38:16,415 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:38:17,886 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:38:18,426 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:38:21,763 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:38:21,770 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:38:21,771 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:38:23,906 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:38:28,924 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:38:33,947 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:38:36,768 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:38:36,773 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:38:36,773 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:38:39,910 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:38:44,923 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:38:46,340 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:38:46,417 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:38:48,033 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:38:50,434 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:38:51,768 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:38:51,775 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:38:51,776 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:38:55,958 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:39:00,981 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:39:06,001 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:39:06,773 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:39:06,781 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:39:06,782 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:39:11,953 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:39:16,346 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:39:16,422 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:39:17,425 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:39:18,171 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:39:21,773 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:39:21,781 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:39:21,782 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:39:22,882 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:39:27,897 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:39:32,926 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:39:36,777 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:39:36,781 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:39:36,782 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:39:38,907 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:39:43,920 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:39:46,352 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:39:46,434 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:39:48,316 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:39:49,447 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:39:51,784 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:39:51,788 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:39:51,788 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:39:54,943 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:39:59,962 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:40:04,987 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:40:06,790 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:40:06,792 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:40:06,792 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:40:10,942 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:40:15,961 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:40:16,361 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:40:16,441 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:40:18,466 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:40:21,463 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:40:21,794 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:40:21,797 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:40:21,797 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:40:26,987 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:40:31,999 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:40:36,797 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:40:36,802 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:40:36,802 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:40:37,956 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:40:42,981 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:40:46,363 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:40:46,447 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:40:48,459 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:40:48,591 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:40:51,801 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:40:51,807 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:40:51,807 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:40:54,001 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:40:59,017 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:41:04,039 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:41:06,803 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:41:06,812 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:41:06,812 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:41:09,042 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:41:14,063 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:41:16,369 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:41:16,447 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:41:18,719 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:41:19,460 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:41:21,811 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:41:21,817 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:41:21,817 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:41:24,967 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:41:29,989 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:41:35,009 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:41:36,817 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:41:36,818 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:41:36,819 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:41:40,951 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:41:45,969 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:41:46,372 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:41:46,444 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:41:48,866 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:41:51,460 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:41:51,820 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:41:51,823 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:41:51,823 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:41:56,954 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:42:01,973 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:42:06,824 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:42:06,826 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:42:06,827 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:42:07,957 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:42:12,972 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:42:16,377 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:42:16,464 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:42:17,999 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:42:18,479 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:42:21,826 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:42:21,828 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:42:21,828 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:42:24,029 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:42:29,049 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:42:34,068 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:42:36,829 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:42:36,829 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:42:36,830 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:42:39,938 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:42:44,960 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:42:46,379 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:42:46,453 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:42:48,126 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:42:50,468 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:42:51,834 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:42:51,835 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:42:51,835 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:42:55,997 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:43:01,015 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:43:06,027 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:43:06,833 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:43:06,835 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:43:06,835 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:43:11,037 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:43:16,056 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:43:16,387 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:43:16,470 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:43:18,263 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:43:21,492 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:43:21,837 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:43:21,839 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:43:21,840 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:43:27,050 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:43:32,073 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:43:36,840 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:43:36,841 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:43:36,843 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:43:37,946 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:43:42,964 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:43:46,395 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:43:46,480 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:43:48,407 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/output.log
2024-01-06 14:43:48,491 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:43:51,846 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:43:51,848 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:43:51,849 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:43:54,032 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:43:59,054 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:44:04,074 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:44:06,848 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: stop_status
2024-01-06 14:44:06,850 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: stop_status
2024-01-06 14:44:06,850 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-06 14:44:09,967 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:44:12,522 DEBUG   SenderThread:62535 [sender.py:send():382] send: telemetry
2024-01-06 14:44:12,529 DEBUG   SenderThread:62535 [sender.py:send():382] send: telemetry
2024-01-06 14:44:12,530 DEBUG   SenderThread:62535 [sender.py:send():382] send: exit
2024-01-06 14:44:12,530 INFO    SenderThread:62535 [sender.py:send_exit():589] handling exit code: 0
2024-01-06 14:44:12,530 INFO    SenderThread:62535 [sender.py:send_exit():591] handling runtime: 956
2024-01-06 14:44:12,530 INFO    SenderThread:62535 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-01-06 14:44:12,531 INFO    SenderThread:62535 [sender.py:send_exit():597] send defer
2024-01-06 14:44:12,531 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: defer
2024-01-06 14:44:12,531 INFO    HandlerThread:62535 [handler.py:handle_request_defer():172] handle defer: 0
2024-01-06 14:44:12,531 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: defer
2024-01-06 14:44:12,531 INFO    SenderThread:62535 [sender.py:send_request_defer():613] handle sender defer: 0
2024-01-06 14:44:12,531 INFO    SenderThread:62535 [sender.py:transition_state():617] send defer: 1
2024-01-06 14:44:12,531 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: defer
2024-01-06 14:44:12,531 INFO    HandlerThread:62535 [handler.py:handle_request_defer():172] handle defer: 1
2024-01-06 14:44:12,531 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: defer
2024-01-06 14:44:12,531 INFO    SenderThread:62535 [sender.py:send_request_defer():613] handle sender defer: 1
2024-01-06 14:44:12,531 INFO    SenderThread:62535 [sender.py:transition_state():617] send defer: 2
2024-01-06 14:44:12,532 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: defer
2024-01-06 14:44:12,532 INFO    HandlerThread:62535 [handler.py:handle_request_defer():172] handle defer: 2
2024-01-06 14:44:12,532 INFO    HandlerThread:62535 [system_monitor.py:finish():203] Stopping system monitor
2024-01-06 14:44:12,532 DEBUG   SystemMonitor:62535 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-01-06 14:44:12,532 INFO    HandlerThread:62535 [interfaces.py:finish():202] Joined cpu monitor
2024-01-06 14:44:12,532 DEBUG   SystemMonitor:62535 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-01-06 14:44:12,532 INFO    HandlerThread:62535 [interfaces.py:finish():202] Joined disk monitor
2024-01-06 14:44:12,533 INFO    HandlerThread:62535 [interfaces.py:finish():202] Joined gpuapple monitor
2024-01-06 14:44:12,533 INFO    HandlerThread:62535 [interfaces.py:finish():202] Joined memory monitor
2024-01-06 14:44:12,533 INFO    HandlerThread:62535 [interfaces.py:finish():202] Joined network monitor
2024-01-06 14:44:12,533 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: defer
2024-01-06 14:44:12,533 INFO    SenderThread:62535 [sender.py:send_request_defer():613] handle sender defer: 2
2024-01-06 14:44:12,533 INFO    SenderThread:62535 [sender.py:transition_state():617] send defer: 3
2024-01-06 14:44:12,533 DEBUG   SenderThread:62535 [sender.py:send():382] send: stats
2024-01-06 14:44:12,533 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: defer
2024-01-06 14:44:12,533 INFO    HandlerThread:62535 [handler.py:handle_request_defer():172] handle defer: 3
2024-01-06 14:44:12,534 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: defer
2024-01-06 14:44:12,534 INFO    SenderThread:62535 [sender.py:send_request_defer():613] handle sender defer: 3
2024-01-06 14:44:12,534 INFO    SenderThread:62535 [sender.py:transition_state():617] send defer: 4
2024-01-06 14:44:12,534 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: defer
2024-01-06 14:44:12,534 INFO    HandlerThread:62535 [handler.py:handle_request_defer():172] handle defer: 4
2024-01-06 14:44:12,534 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: defer
2024-01-06 14:44:12,534 INFO    SenderThread:62535 [sender.py:send_request_defer():613] handle sender defer: 4
2024-01-06 14:44:12,534 INFO    SenderThread:62535 [sender.py:transition_state():617] send defer: 5
2024-01-06 14:44:12,534 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: defer
2024-01-06 14:44:12,534 INFO    HandlerThread:62535 [handler.py:handle_request_defer():172] handle defer: 5
2024-01-06 14:44:12,534 DEBUG   SenderThread:62535 [sender.py:send():382] send: summary
2024-01-06 14:44:12,534 INFO    SenderThread:62535 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-01-06 14:44:12,535 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: defer
2024-01-06 14:44:12,535 INFO    SenderThread:62535 [sender.py:send_request_defer():613] handle sender defer: 5
2024-01-06 14:44:12,535 INFO    SenderThread:62535 [sender.py:transition_state():617] send defer: 6
2024-01-06 14:44:12,535 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: defer
2024-01-06 14:44:12,535 INFO    HandlerThread:62535 [handler.py:handle_request_defer():172] handle defer: 6
2024-01-06 14:44:12,535 DEBUG   SenderThread:62535 [sender.py:send_request():409] send_request: defer
2024-01-06 14:44:12,535 INFO    SenderThread:62535 [sender.py:send_request_defer():613] handle sender defer: 6
2024-01-06 14:44:12,538 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: status_report
2024-01-06 14:44:12,615 ERROR   SenderThread:62535 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-06 14:44:12,615 ERROR   SenderThread:62535 [internal_api.py:execute():374] {"errors":[{"message":"run curtis-pond/marketing-analysis/wf6xw9ej was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-06 14:44:13,523 INFO    Thread-25 :62535 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/files/wandb-summary.json
2024-01-06 14:44:13,535 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-06 14:44:13,743 ERROR   SenderThread:62535 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-06 14:44:13,743 ERROR   SenderThread:62535 [internal_api.py:execute():374] {"errors":[{"message":"run curtis-pond/marketing-analysis/wf6xw9ej was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-06 14:44:15,997 ERROR   SenderThread:62535 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-06 14:44:15,997 ERROR   SenderThread:62535 [internal_api.py:execute():374] {"errors":[{"message":"run curtis-pond/marketing-analysis/wf6xw9ej was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-06 14:44:15,997 INFO    SenderThread:62535 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 369, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql
2024-01-06 14:44:18,554 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:44:20,424 ERROR   SenderThread:62535 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-06 14:44:20,424 ERROR   SenderThread:62535 [internal_api.py:execute():374] {"errors":[{"message":"run curtis-pond/marketing-analysis/wf6xw9ej was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-06 14:44:23,575 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:44:28,594 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:44:28,825 ERROR   SenderThread:62535 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-06 14:44:28,825 ERROR   SenderThread:62535 [internal_api.py:execute():374] {"errors":[{"message":"run curtis-pond/marketing-analysis/wf6xw9ej was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-06 14:44:33,617 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:44:38,636 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:44:42,616 ERROR   FileStreamThread:62535 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/curtis-pond/marketing-analysis/wf6xw9ej/file_stream
NoneType: None
2024-01-06 14:44:43,651 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:44:48,667 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:44:48,904 ERROR   SenderThread:62535 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-06 14:44:48,904 ERROR   SenderThread:62535 [internal_api.py:execute():374] {"errors":[{"message":"run curtis-pond/marketing-analysis/wf6xw9ej was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-06 14:44:53,684 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:44:58,697 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:45:03,715 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:45:08,734 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:45:13,754 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:45:18,770 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:45:23,791 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:45:28,148 ERROR   SenderThread:62535 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-06 14:45:28,148 ERROR   SenderThread:62535 [internal_api.py:execute():374] {"errors":[{"message":"run curtis-pond/marketing-analysis/wf6xw9ej was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-06 14:45:28,809 DEBUG   HandlerThread:62535 [handler.py:handle_request():146] handle_request: keepalive
2024-01-06 14:46:35,008 ERROR   SenderThread:62535 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-06 14:46:35,010 ERROR   SenderThread:62535 [internal_api.py:execute():374] {"errors":[{"message":"run curtis-pond/marketing-analysis/wf6xw9ej was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-06 14:46:35,041 INFO    HandlerThread:62535 [handler.py:finish():866] shutting down handler
2024-01-06 14:46:35,077 INFO    WriterThread:62535 [datastore.py:close():294] close: /Users/curtispond/Documents/wandb/wandb/run-20240106_142810-wf6xw9ej/run-wf6xw9ej.wandb
2024-01-06 14:46:35,869 ERROR   StreamThr :62535 [internal.py:wandb_internal():174] Thread SenderThread:
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2212, in upsert_run
    response = self.gql(
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 341, in gql
    ret = self._retry_gql(
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 369, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 49, in run
    self._run()
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 100, in _run
    self._process(record)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/internal.py", line 328, in _process
    self._sm.send(record)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/sender.py", line 389, in send
    send_handler(record)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/sender.py", line 411, in send_request
    send_handler(record)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/sender.py", line 639, in send_request_defer
    self.debounce(final=True)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/sender.py", line 540, in debounce
    self._maybe_update_config(always=final)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/sender.py", line 517, in _maybe_update_config
    self._debounce_config()
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/sdk/internal/sender.py", line 546, in _debounce_config
    self._api.upsert_run(
  File "/opt/homebrew/Caskroom/miniforge/base/envs/cx-analytics/lib/python3.9/site-packages/wandb/apis/normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: run curtis-pond/marketing-analysis/wf6xw9ej was previously created and deleted; try a new run name (Error 409: Conflict)
